example/package.json

```diff
@@ -7,8 +7,8 @@
     "sync": "rm -rf ./node_modules/react-native-gifted-chat; sane '/usr/bin/rsync -v -a --exclude .git --exclude example --exclude __tests__ --exclude node_modules ../ ./node_modules/react-native-gifted-chat/' .. --glob='{**/*.json,**/*.js}'"
   },
   "dependencies": {
-    "react": "15.2.1",
-    "react-native": "^0.30.0",
+    "react": "~15.4.0-rc.4",
```

Comment:
RN has the dependency on that specific react version. I tried to use the latest React version 15.4.2 but it does not work. I think when they release the latest RN to npm they'll update that too. I hope so